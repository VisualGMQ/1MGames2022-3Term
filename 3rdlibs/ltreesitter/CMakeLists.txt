aux_source_directory(./csrc LTREESITTER_SRC)
add_library(ltreesitter SHARED ${LTREESITTER_SRC})
target_include_directories(ltreesitter PRIVATE ./include)
target_compile_features(ltreesitter PRIVATE c_std_99)
target_link_libraries(ltreesitter PRIVATE lua)
target_link_libraries(ltreesitter PRIVATE treesitter)

if (UNIX)
    target_link_libraries(ltreesitter PRIVATE dl)
endif()

add_subdirectory(./vendors/cpp)