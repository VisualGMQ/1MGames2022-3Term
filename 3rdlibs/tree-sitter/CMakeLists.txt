aux_source_directory(./src TREESITTER_SRC)
aux_source_directory(./src/unicode TREESITTER_SRC)
add_library(treesitter STATIC ${TREESITTER_SRC})
target_include_directories(treesitter PUBLIC ./include)
target_include_directories(treesitter PRIVATE ./src)
target_compile_features(treesitter PRIVATE c_std_99)