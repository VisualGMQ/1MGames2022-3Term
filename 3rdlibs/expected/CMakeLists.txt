add_library(expected INTERFACE)
target_include_directories(expected INTERFACE ./include)